.travis-script.sh

```diff
@@ -1,48 +1,56 @@
-sudo apt-get --yes update
-sudo apt-get --yes install python-pip
-sudo pip install enum34
-sudo pip install nose
+# All apt-get installs
+sudo apt-get --yes update # Update
+sudo apt-get --yes install python-pip # Installing pip
+sudo apt-get --yes install wget # Installing wget
+sudo apt-get --yes install default-jdk # Install Java JDK
+
+# Install required packages
+sudo pip install -r /app/requirements.txt
```

Comment:
What does this install?

Reply:
Ok I see, enum34 and nose, should this change be in a separate commit?

Reply:
@etejedor , Yup. That'd be better. I'll make a different commit for that 👍

Reply:
@etejedor , any other changes ? If not, I'll rebase commits into 3, including the one for this one (`requirements.txt`)